hwdetect/hwdetect.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# hwdetector: detects connected hardware.
+
+detect_hardware()
```

Comment:
Where is the body of script detect_hardware?

Reply:
Fixed